pymc3/plots/compareplot.py

```diff
@@ -11,9 +12,22 @@ def compareplot(comp_df, ax=None):
     ----------
 
     comp_df: DataFrame
-        The result of the pm.compare() function
+        the result of the `pm.compare()` function
+    insample : bool
+        plot the in-sample deviance, that is the value of the IC without the
+        penalization given by the effective number of parameters (pIC).
+        Defaults to True
+    se : bool
+        plot the standard error of the IC estimate. Defaults to True
+    dse : bool
```

Comment:
`diff_se` (or similar) would be clearer

Reply:
I'm using `se` and `dse` because those names are the columns names in `comp_df`. Still want me to change it?

Reply:
These are only suggestions. Feel free to ignore if they don't make sense.